identities/models.py

```diff
@@ -44,3 +48,74 @@ class Identity(models.Model):
 
     def __str__(self):
         return str(self.id)
+
+    def remove_details(self, user):
+        updated_details = {}
+        for attribute, value in self.details.iteritems():
```

Comment:
The `iteritems` here and elsewhere are causing the Python 3 test failures. In Python 3, the `.items()` method does what `iteritems()` did in Python 2, and there is no longer an `iteritems` method. I think just use `.items()` here -- it's an extra temporary list in Python 2, but hopefully `details` remains of a reasonable size. :)